jobs/build/promote/Jenkinsfile

```diff
@@ -333,6 +333,20 @@ node {
                     errata_url = retval.errataUrl
                 }
             }
+            stage("add cve flaw bugs") {
+                if (params.DRY_RUN || advisory == -1) {
+                    return
+                }
+                if (arch != 'x86_64') {
+                    return
+                }
```

Comment:
This will require us to promote x86_64 first then other arches, right?
I guess we can remove this check since it will be no-ops if flaw bugs are already attached.